### PR TITLE
Add endpoints for clearing and generating data

### PR DESCRIPTION
**Pull request by BabarKhanDev**
This will be useful in the demonstration. It does the same thing as running the data generation script that already existed, but now we can put that functionality into a ui

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -117,7 +118,7 @@ async def upload_image(file: UploadFile = File(...)):
 
 
 @app.get("/track_growth")
-async def track_growth(center: Tuple[float, float] = (0, 0), scan_range: float = 360, day_range: int = 1000):
+async def track_growth(center: Tuple[float, float] = (0, 0), scan_range: float = 360, day_range: int = 1000) :
```
Comment (Laura7089) [inline app/main.py:121]: Errant space spotted

File: util/data_generators.py [hunk 2]
```diff
@@ -45,31 +45,47 @@ def generate_entry(conn, date: datetime, plant_id: str, location_center=(54.39,
     insert_processed_entry(conn, processed_entry)
 
 
+def insert_data(totals_knotweed, totals_rose, conn):
+    days = list(range(0, 360))
```
Comment (Laura7089) [inline util/data_generators.py:49]: I don't think this needs to be a list - in fact, can you define inline in the below list comp?

File: util/data_generators.py [hunk 3]
```diff
@@ -45,31 +45,47 @@ def generate_entry(conn, date: datetime, plant_id: str, location_center=(54.39,
     insert_processed_entry(conn, processed_entry)
 
 
+def insert_data(totals_knotweed, totals_rose, conn):
```
Comment (Laura7089) [inline util/data_generators.py:48]: Optional: type annotations

File: util/data_generators.py [hunk 4]
```diff
@@ -45,31 +45,47 @@ def generate_entry(conn, date: datetime, plant_id: str, location_center=(54.39,
...
+            generate_entry(conn, date, plant_id="Knotweed")
+
+    for date, rose_count in zip(datetime_days, totals_rose):
+        for _ in range(math.floor(rose_count)):
+            generate_entry(conn, date, plant_id="Rose")
+
+
+def clear_data(conn):
+    with conn.cursor() as cursor:
+        cursor.execute("""truncate image_processing.processed_entry, image_processing.raw_entry""")
```
Comment (Laura7089) [inline util/data_generators.py:63]: Triple quotes are (I believe) unnecessary

Comment (Laura7089) [reply util/data_generators.py:63]: Also, does this need to be `TRUNCATE TABLE` or am I misunderstanding?

**Review by Laura7089 — CHANGES_REQUESTED** [review]
One or two small things

File: app/main.py (hunk 1 above)
Comment (BabarKhanDev) [reply app/main.py:121]: fixed :)

File: util/data_generators.py (hunk 2 above)
Comment (BabarKhanDev) [reply util/data_generators.py:49]: oops, artefact from earlier version. Fixed ty :)

**Review by Laura7089 — APPROVED** [review]

**Pull request merged**